code/datums/quick_load_outfits.dm

```diff
@@ -1141,6 +1141,29 @@
 	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/som, SLOT_IN_ACCESSORY)
 	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary/som, SLOT_IN_ACCESSORY)
 
+/datum/outfit/quick/som/marine/light_carbine
+	name = "V-34 Light infantryman"
```

Comment:
```suggestion
	name = "V-34 Light Infantryman"
```

Reply:
I probs have formatted this way across the whole file. I should probably change it though.